client-js/main/client/backend-client.js

```diff
@@ -387,6 +397,29 @@ export class BackendClient {
     return new FirebaseBackendClient(endpoint, firebaseClient, requestFactory, subscriptionService);
   }
 
+
+  /**
+   * Registers all Protobuf types provided by the specified modules.
+   *
+   * After the registration, the types can be used and parsed correctly.
+   *
+   * @param protoIndexFiles the index.js files generated by Spine ProtoJsPlugin
```

Comment:
Do we call it `ProtoJsPlugin`? Or is it a class? Please review this particular naming here and below.